lib/post.rb

```diff
@@ -0,0 +1,29 @@
+require 'active_record'
+
+if ENV['ENVIRONMENT'] == 'test'
+  ActiveRecord::Base.establish_connection(
+    adapter: 'postgresql',
+    host: 'localhost',
+    username: ENV["USERNAME"],
+    password: ENV["SQL_PASSWORD"],
+    database: 'chitter_test'
+  )
+else 
+  ActiveRecord::Base.establish_connection(
+    adapter: 'postgresql',
+    host: 'localhost',
+    username: ENV["USERNAME"],
+    password: ENV["SQL_PASSWORD"],
+    database: 'chitter'
+  )
+end
+
+class Post < ActiveRecord::Base
+  self.table_name = 'posts'
```

Comment:
By default, AR checks for a table than is the lowercase and pluralised version of the class name, and a primary key column named `id`, which means you can probably remove these two lines of setup as they describe the default behaviour

Reply:
I had left these in as a precaution but you're right, I shouldn't be keeping unnecessary code